redbot/cogs/admin/announcer.py

```diff
@@ -38,8 +39,8 @@ def cancel(self):
         """
         self.active = False
 
-    async def _get_announce_channel(self, guild: discord.Guild) -> discord.TextChannel:
-        if await ctx.bot.cog_disabled_in_guild_raw("Announcer", guild.id):
+    async def _get_announce_channel(self, guild: discord.Guild) -> Optional[discord.TextChannel]:
+        if await ctx.bot.cog_disabled_in_guild_raw("Admin", guild.id):
```

Comment:
ctx is not avaliable here.

Reply:
Holding off on changing this till whatever announcer changes happen since they will likely rquire me to rebase this PR